notebooks/AC_and_CS.ipynb

```diff
@@ -0,0 +1,200 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import votekit.ballot_generator as bg"
```

Comment:
Let's move this line `import votekit.ballot_generator as bg` down to first big code chunk just for cohesiveness